lib/improver/utilities/cube_metadata.py

```diff
@@ -37,6 +37,47 @@
 
 from improver.utilities.cube_manipulation import compare_coords
 
+# Define correct v1.2.0 meta-data for v1.1.0 data.
+GRID_ID_LOOKUP = {'enukx_standard_v1': {'mosg__grid_type': 'standard',
+                                        'mosg__model_configuration': 'uk_ens',
+                                        'mosg__grid_domain': 'uk_extended',
+                                        'mosg__grid_version': '1.1.0'},
+                  'engl_standard_v1': {'mosg__grid_type': 'standard',
+                                       'mosg__model_configuration': 'gl_ens',
+                                       'mosg__grid_domain': 'global',
+                                       'mosg__grid_version': '1.1.0'},
+                  'ukvx_standard_v1': {'mosg__grid_type': 'standard',
+                                       'mosg__model_configuration': 'uk_det',
+                                       'mosg__grid_domain': 'uk_extended',
+                                       'mosg__grid_version': '1.1.0'},
+                  'glm_standard_v1': {'mosg__grid_type': 'standard',
+                                      'mosg__model_configuration': 'gl_det',
+                                      'mosg__grid_domain': 'global',
+                                      'mosg__grid_version': '1.1.0'},
+                  }
+
+
+def stage_v110_to_v120(cube):
+    """Translates meta-data from StaGE version 1.1.0 to StaGE version 1.2.0
+    Cubes that have no "grid_id" attribute are not recognised as v1.1.0 and
+    are ignored.
```

Comment:
Suggest `Translates grid-describing metadata` or `grid-describing attributes` here, to be more specific about the change from 1.1.0 to 1.2.0.

Reply:
Doc-string improved.